.github/workflows/acceptance.yml

```diff
@@ -251,6 +251,152 @@ jobs:
+
+        # Install node
+        - uses: volta-cli/action@v1
+
+        # Yarn
+        #   - See https://github.com/actions/cache/blob/master/examples.md#node---yarn for example
+        # Python
+        #   Use `.python-version` to avoid duplication
+        #   XXX: can't actually read from .python-version because GitHub Actions
+        #   does not support our version (2.7.16)
+        #
+        #   XXX: Using `2.7` as GHA image only seems to keep one minor version around and will break
+        #   CI if we pin it to a specific patch version.
+        - name: Set up outputs
+          id: config
+          env:
+            MATRIX_INSTANCE: ${{ matrix.instance }}
+          run: |
+            echo "::set-output name=yarn-cache-dir::$(yarn cache dir)"
+            echo "::set-output name=python-version::3.6.12"
+            echo "::set-output name=matrix-instance-number::$(($MATRIX_INSTANCE+1))"
+            echo "::set-output name=acceptance-dir::.artifacts/visual-snapshots/acceptance"
+
+        # yarn cache
+        - uses: actions/cache@v1
+          id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+          with:
+            path: ${{ steps.config.outputs.yarn-cache-dir }}
+            key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+            restore-keys: |
+              ${{ runner.os }}-yarn-
+
+        # setup python
+        - name: Set up Python ${{ steps.config.outputs.python-version }}
+          uses: actions/setup-python@v1
+          with:
+            python-version: ${{ steps.config.outputs.python-version}}
+
+        # setup pip
+        - name: Install pip
+          run: |
+            pip install --no-cache-dir --upgrade "pip>=20.0.2"
+
+        # pip cache
+        - name: Get pip cache dir
+          id: pip-cache
+          run: |
+            echo "::set-output name=dir::$(pip cache dir)"
+
+        - name: pip cache
+          uses: actions/cache@v1
+          with:
+            path: ${{ steps.pip-cache.outputs.dir }}
+            key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements-*.txt') }}
+            restore-keys: |
+              ${{ runner.os }}-pip-
+
+        - name: Install Javascript Dependencies
+          run: |
+            yarn install --frozen-lockfile
+
+        - name: Install Python Dependencies
+          env:
+            PGPASSWORD: postgres
+          run: |
+            python setup.py install_egg_info
+            pip install wheel # GitHub Actions does not have this installed by default (unlike Travis)
+            pip install -U -e ".[dev]"
+            pip uninstall -y rb
+            pip install -e git+https://github.com/getsentry/rb.git@master#egg=rb
+
+        - name: Start devservices
+          run: |
+            sentry init
+            sentry devservices up postgres redis clickhouse snuba
+
+        - name: webpack
+          run: |
+            yarn webpack --display errors-only
```

Comment:
This could be omitted if `SENTRY_LIGHT_BUILD` is unset, and any webpacking errors would fail earlier, before devservices.

Reply:
Oh, should do that for 2.7 acceptance too. I might have missed that in earlier review, I don't recall.

Reply:
Original PR https://github.com/getsentry/sentry/pull/18576, looks like I didn't complain about this before.

Reply:
After some offline discussion, ignore what I said above. It's useful to have yarn webpack explicitly so time can be measured. And also I want to remove SENTRY_LIGHT_BUILD and make it the default in the future.